gallery/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -375,10 +375,10 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
-				CODE_SIGN_IDENTITY = "Apple Development";
-				CODE_SIGN_STYLE = Automatic;
+				CODE_SIGN_IDENTITY = "iPhone Developer";
```

Comment:
This line (and 2 other `CODE_SIGN_IDENTITY = "iPhone Developer";`) were not in the original file. Why are they added?

Reply:
Not sure, but works without. Removed so its identical as before